renovate.json5

```diff
@@ -9,29 +9,5 @@
       groupName: 'netlify packages',
       schedule: null,
     },
-    {
-      // Those cannot be upgraded to a major version until we drop support for Node 8
-      packageNames: [
-        '@octokit/rest',
-        'ava',
-        'chalk',
-        'concordance',
-        'cosmiconfig',
-        'del',
-        'dot-prop',
-        'execa',
-        'find-up',
-        'globby',
-        'locate-path',
-        'log-symbols',
-        'npm-packlist',
-        'ora',
-        'wrap-ansi',
-        'update-notifier',
```

Comment:
It would be nice to follow up with upgrading those dependencies as well.

Reply:
Yes, I'm counting on renovate to open those PRs so I can examine the changelogs for breaking changes (there might ones not related to the Node.js upgrade).